example/transaction/knaken.ts

```diff
@@ -0,0 +1,19 @@
+import buckarooClient from '../buckarooClient';
+
+const knaken = buckarooClient.method('knaken');
+
+//Pay
+knaken
+    .pay({
+        amountDebit: 10.1,
+        description: 'Knaken Payment',
+    })
+    .request();
+//Refund
+knaken
+    .refund({
+        originalTransactionKey: '',
```

Comment:
place an 'X' to indicate the transaction format, so the user will know how the format should be